codec/fuzz/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "commonware-codec-fuzz"
-version = "0.0.1"
+version = "0.0.55"
```

Comment:
Opted to bump the versions for the fuzz crates to keep consistent with everything else (although not strictly necessary) 👍